main.go

```diff
@@ -134,6 +135,14 @@ func main() {
 		setupLog.Error(err, "unable to create controller", "controller", "SwiftRing")
 		os.Exit(1)
 	}
+	if err = (&controllers.SwiftReconciler{
+		Client: mgr.GetClient(),
+		Scheme: mgr.GetScheme(),
+		Log:    mgr.GetLogger(),
```

Comment:
Should probably initialize `Kclient` here as well

Reply:
Thanks, missed that - fixed!